content/en/docs/contribute/style/diagram-guide.md

```diff
@@ -0,0 +1,447 @@
+
+Generate an `.svg` or `.png` file, download this file to the appropriate `../images/` folder, and then reference the `.svg` or `.png` file using a markdown image statement.
+
+Here is an example markdown image statement for `your-image.svg`. 
+
+```json
+![your-image.svg](static/images/boxnet.svg)
+```
+You will note that the `.svg` file handling is the same for the `Mermaid+SVG` and `External Tool` methods. 
+
+{{< mermaid >}}
+graph LR
+A[1. Use external<br>tool to create/edit<br>diagram] -->
+B[2. If possible, save<br>diagram coordinates<br>for contributor<br>access] -->
+C[3. Generate .svg <br>or.png file<br>and download to<br>appropriate<br>images/ folder] -->
+D[4. Reference .svg or .png file<br>using markdown image<br>statement in page .md file page]
+
+
+    classDef box fill:#fff,stroke:#000,stroke-width:1px,color:#000;
+    class A,B,C,D,E box
+{{< /mermaid >}}
+
+<figure style="padding-left: 310px; font-weight: bold" alt="diagram guide external tool method"> 
+<figcaption><a href="https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb2RlIjoiZ3JhcGggTFJcbiAgICBBWzEuIFVzZSBleHRlcm5hbDxicj50b29sIHRvIGNyZWF0ZS9lZGl0PGJyPmRpYWdyYW1dIC0tPlxuICAgIEJbMi4gSWYgcG9zc2libGUsIHNhdmU8YnI-ZGlhZ3JhbSBjb29yZGluYXRlczxicj5mb3IgY29udHJpYnV0b3I8YnI-YWNjZXNzXSAtLT5cbiAgICBDWzMuIEdlbmVyYXRlIC5zdmcgPGJyPm9yLnBuZyBmaWxlPGJyPmFuZCBkb3dubG9hZCB0bzxicj5hcHByb3ByaWF0ZTxicj5pbWFnZXMvIGZvbGRlcl0gLS0-XG4gICAgRFs0LiBSZWZlcmVuY2UgLnN2ZyBvciAucG5nIGZpbGU8YnI-dXNpbmcgbWFya2Rvd24gaW1hZ2U8YnI-c3RhdGVtZW50IGluIHBhZ2UgLm1kIGZpbGUgcGFnZV1cblxuXG4gICAgY2xhc3NEZWYgYm94IGZpbGw6I2ZmZixzdHJva2U6IzAwMCxzdHJva2Utd2lkdGg6MXB4LGNvbG9yOiMwMDA7XG4gICAgY2xhc3MgQSxCLEMsRCxFIGJveFxuICAgICIsIm1lcm1haWQiOiJ7XG4gIFwidGhlbWVcIjogXCJkZWZhdWx0XCJcbn0iLCJ1cGRhdGVFZGl0b3IiOmZhbHNlLCJhdXRvU3luYyI6dHJ1ZSwidXBkYXRlRGlhZ3JhbSI6ZmFsc2V9">Figure x. External Tool method steps</a></figcaption><br>
+</figure>
+
+If your external drawing tool permits:
+
+* You can incorporate multiple `.svg` or `.png` logos, icons and images into your diagram. However, make sure you follow K8s/website guidelines on the use of [third party content](https://kubernetes.io/docs/contribute/style/content-guide/).
+* You should save the diagram coordinates for contributor access. For example, your tool may offer a link to the diagram image, or you could place the source code file, such as an `.xml` file, somewhere for contributor access.
+
+For more information on K8s and CNCF logos and images, check out [CNCF Artwork](https://github.com/cncf/artwork).
+
+The advantages of the external tool method consist of the following: 
+
+* Contributor familiarity with external tool.
+* Diagrams require more detail than what Mermaid can offer.
+
+Don't forget to check the rendered diagram using the local and Netlify previews.
```

Comment:
nit: maybe also link to https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally